utils/validateAndStatusLinks.js

```diff
@@ -0,0 +1,89 @@
+const fetch = require('node-fetch');
+const chalk = require('chalk')
+
+const linksFunctions = {};
```

Comment:
recomiendo usar let para las variables que mutan los valores
```suggestion
let linksFunctions = {};
```